netstandard/ref/System.Drawing.cs

```diff
@@ -70,6 +70,7 @@ public partial struct Color
         public static System.Drawing.Color Indigo { get { throw null; } }
         public bool IsEmpty { get { throw null; } }
         public bool IsNamedColor { get { throw null; } }
+        public bool IsSystemColor { get { throw null; } }
```

Comment:
Does this have a consistent cross-platform meaning?

Reply:
It is well-defined in the sense that it indicates whether the specific `Color` represents a color created from `FromName(String)` and `FromKnownColor(KnownColor)`. However, not all values of `KnownColor` will be meaningful everywhere.

Reply:
Seems good enough to me :)